docs/docs/GettingStarted/query_examples.md

```diff
@@ -0,0 +1,150 @@
+# Query Examples
```

Comment:
This page was more an internal brainstorming rather than a public facing documentation, I would not include this in the public documentation or if we want this needs to be polished/cleaned. This mostly consists of random thoughts for now.

Reply:
I removed the file from navigation but kept it in the docs, we can now accept this PR as is, and later think about whether or not we want to publish this Queries. But I think we have to look over them with the original authors and ensure that they are actually working